resources/authgear/templates/en/web/authflowv2/enter_password.html

```diff
@@ -39,12 +39,14 @@ <h1 class="screen-title">
     {{ end }}
     <!-- TODO: Handle button loading state in https://github.com/authgear/authgear-server/issues/3676 -->
     <button
+      form="main-form"
       class="primary-btn w-full"
       type="submit"
       name="x_action"
       value=""
       data-authgear-event="authgear.button.enter_password"
       data-action-button
+      data-loading-state
```

Comment:
Lets name this attribute `data-loadable`, which seems clearer it configures the button as a loadable button.